.github/workflows/azure_tests.yml

```diff
@@ -0,0 +1,18 @@
+name: Azure unit tests
+
+on: [pull_request]
+
+jobs:
+    build:
+      runs-on: ubuntu-latest
+      steps:
+        - uses: actions/checkout@v4
+        - name: Use Node.js
+          uses: actions/setup-node@v3
+          with:
+            node-version: '18.x'
+        - run: npm install
+          working-directory: azure
+        - run: npm run test
+          working-directory: azure
+  
```

Comment:
why did this PR require a new workflow for running tests. Wouldn't the existing workflows run the tests?

Reply:
During the PR process found there was no existing workflow that ran the test.